cognite/extractorutils/extraction_pipelines.py

```diff
@@ -0,0 +1,116 @@
+#  Copyright 2022 Cognite AS
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import logging
+from functools import wraps
+from threading import Event, Thread
+
+from cognite.client import CogniteClient
+from cognite.client.data_classes import ExtractionPipelineRun
+
+
+def add_extraction_pipeline(
+    extraction_pipeline_ext_id: str,
+    cognite_client: CogniteClient,
+    heartbeat_waiting_time: int = 5,
```

Comment:
This is very short, in the base class it is set to `600` (ie, 10 minutes). Any reason it is set so low here? I think having the same default across the framework is nice.

Keep in mind that the backend will not check for missing heartbeats more than once every 15 minutes, so this will not make a huge difference in responsiveness

Reply:
``` suggestion
    heartbeat_waiting_time: int = 600,
```

Reply:
Sorry for missing this the first time :)